modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-consume.adoc

```diff
@@ -0,0 +1,92 @@
+= Anypoint MQ Consume Operation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The `consume` operation in the Anypoint MQ connector provides the ability to consume a single message at any time of the flow, from any destination, using the acknowledgment strategy that supports your use case.
+
+== Incoming Message Structure
+
+Every consumed message consists of two parts:
+
+* `payload` - Represents the value of the received message body
```

Comment:
APOLOGIES for previous comments about "payload": STET.

Do we need "value of the received message body"? How about just "message body" or "message content"?